dist/debian/control.template

```diff
@@ -7,103 +7,66 @@ X-Python3-Version: >= 3.4
 Standards-Version: 3.9.5
 Rules-Requires-Root: no
 
-Package: %{product}-conf
+Package: scylla-conf
 Architecture: any
+Provides: scylla-enterprise-conf
+Replaces: scylla-server (<< 1.1), scylla-enterprise-conf (<< 2025.1.0~)
+Conflicts: scylla-server (<< 1.1), scylla-enterprise-conf (<< 2025.1.0~)
+Breaks: scylla-enterprise-conf (<< 2025.1.0~)
```

Comment:
What changed in this line?

Reply:
nothing, it was just moved to other lines, i will revert it